docs/quickstart-rosa.md

```diff
@@ -226,6 +234,12 @@ Deleting a ROSA cluster consists of two parts
   ```bash
   rosa delete cluster --cluster=<cluster-name>
   ```
+   Delete Cluster's operator-roles and oidc-provider as shown in the above delete cluster command's output. For e.g.
+  
+  ```bash
+  rosa delete operator-roles -c <cluster-name>
```

Comment:
this is needed though :)

Reply:
Changes are in [kumuduh-patch-6](https://github.com/rh-mobb/documentation/tree/kumuduh-patch-6). Thanks !